src/modules/infobox/components/FeatureElevationProfilePlot.vue

```diff
@@ -105,238 +122,304 @@ export default {
+            return round(
+                (this.elevationProfile.maxElevation - this.elevationProfile.minElevation) / 10.0
+            )
+        },
+        /**
+         * Defines a buffer of 10% of the elevation delta (with a minimum of 5m), so that there is
+         * always some plot/chart below the lowest point (it will not be glued to the X axis line,
+         * but drawn a bit above)
+         */
+        yAxisMinimumValue() {
+            return Math.max(
+                Math.floor(this.elevationProfile.minElevation) -
+                    Math.max(Math.floor(this.tenPercentOfElevationDelta), 5),
+                0
+            )
+        },
+        /**
+         * Same things as the minimum value, will give a 10% of the elevation delta as buffer, with
+         * a minimum of 5meters
+         */
+        yAxisMaxValue() {
+            return (
+                Math.ceil(this.elevationProfile.maxElevation) +
+                Math.max(Math.ceil(this.tenPercentOfElevationDelta), 5)
+            )
+        },
+        /** Definition of the data ChartJS will show, with some styling configuration too */
+        chartJsData() {
+            return {
+                datasets: [
+                    {
+                        label: `${this.$t('elevation')}`,
+                        data: this.elevationProfile.points,
+                        parsing: {
+                            xAxisKey: 'dist',
+                            yAxisKey: 'elevation',
+                        },
+                        pointRadius: 1,
+                        pointHoverRadius: 3,
+                        borderColor: 'rgb(255, 99, 132)',
+                        borderWidth: 1,
+                        fill: {
+                            target: 'origin',
+                            above: 'rgba(255, 99, 132, 0.7)',
+                        },
+                        // smooth up a bit the line (can be removed/reverted to 'default' if we want a sharper line)
+                        cubicInterpolationMode: 'monotone',
+                    },
+                ],
+            }
+        },
+        /** Definition of axis for the profile chart */
+        chartJsScalesConfiguration() {
+            return {
+                x: {
+                    type: 'linear',
+                    max: this.elevationProfile.maxDist,
+                    title: {
+                        display: true,
+                        text: `${this.$t('distance_label')} [${this.unitUsedOnDistanceAxis}]`,
+                        font: {
+                            weight: 'bold',
+                        },
+                        // removing the padding so that we gain a bit of vertical space
+                        padding: 0,
+                    },
+                    ticks: {
+                        // processing distance number to be more human-readable
+                        callback: (val) => round(val * this.factorToUseForDisplayedDistances, 1),
```

Comment:
Not sure that returning a callback function in computed property is a good idea.

Reply:
I don't know either. I don't think we can externalise that (that's a requirement from ChartJS to give a function to process this styling) as we are using some other computed/data props here (`this.factorToUseForDisplayedDistance`)